### PR TITLE
Add support for 32/64-bit ARM/Linux targets

### PR DESCRIPTION
**Pull request by dsobotta**
arm32 tested on Raspberry Pi Zero W/Raspbian
arm64 tested on QEMU/Debian

Capabilities - #67

### CODE REVIEW

File: README_OTHERPLATFORMS.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green)| Toit generate boot snapshot |
 | ![](https://img.shields.io/static/v1?label=&message=SUCCESS&color=green) | Toit run examples |
-| ![](https://img.shields.io/static/v1?label=&message=PARTIAL&color=yellow) | Cross-compile for RISC-V > [fails to link on Arch](https://github.com/dsobotta/toit-riscv/issues/6)|
-| ![](https://img.shields.io/static/v1?label=&message=TODO&color=orange) | Embedded RISC-V support |
+| ![](https://img.shields.io/static/v1?label=&message=PARTIAL&color=yellow) | Cross-compile > [fails to link on Arch](https://github.com/dsobotta/toit/issues/6)|
+| ![](https://img.shields.io/static/v1?label=&message=TODO&color=orange) | Embedded support |
 
 
-## 1) RISC-V Environment Setup
+## 1) Environment Setup (RISC-V Example)
```
Comment (kasperl) [inline README_OTHERPLATFORMS.md:17]: Nit: Stick to non-"title case" for all these (for consistency with other titles and the main README)?

**Comment by erikcorry** [comment]
Don't you need the esp-idf branch for RISC-V any more?

**Comment by dsobotta** [comment]
> Don't you need the esp-idf branch for RISC-V any more?

Only if attempting to build for ESP32 from a RISC-V/ARM32/ARM64 host.  There are a few lines to support this in my fork of esp-idf, but I'm going to submit those separately (as the goals are different, and separate repo too...).  For now I thought it best to avoid confusion and remove references to a potentially out-of-date fork.

**Comment by erikcorry** [comment]
The sandbox file is getting messy but we can clean that up later.  LGTM.

**Pull request merged**